presto-docs/src/main/sphinx/release/release-0.242.rst

```diff
@@ -0,0 +1,52 @@
+=============
+Release 0.242
+=============
+
+General Changes
+_______________
+* Fix compiler error in LambdaBytecodeGenerator when CSE is enabled and multiple SQL functions contain lambda expressions.
```

Comment:
My bad, let's change "when CSE is enabled" to "when common sub-expression optimization is enabled"